mllib/src/test/scala/org/apache/spark/ml/util/MLTestingUtils.scala

```diff
@@ -27,4 +31,56 @@ object MLTestingUtils {
     assert(copied.parent.uid == model.parent.uid)
     assert(copied.parent == model.parent)
   }
+
+  def genClassifDFWithNumericLabelCol(
+    sqlContext: SQLContext,
+    labelColName: String,
+    featuresColName: String
+  ): Map[NumericType, DataFrame] = {
```

Comment:
nit: Spark style seems to be to put the return type on the line above, if it will fit. So:

```scala
def genClassifDFWithNumericLabelCol(
    sqlContext: SQLContext,
    labelColName: String,
    featuresColName: String): Map[NumericType, DataFrame] = {
```
[Example](https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/mllib/tree/GradientBoostedTrees.scala#L168)